{{ cookiecutter.package_name|slugify }}/.github/workflows/test.yml

```diff
@@ -9,48 +9,46 @@ on:
 
 jobs:
 
-  test:
-
+  build:
     runs-on: ubuntu-latest
-
-    strategy:
-
-      matrix:
-        python-version: ["{{ cookiecutter.python_version }}"]
-      fail-fast: false
-
     steps:
-
       - name: Checkout
         uses: actions/checkout@v3
-
-      - name: Set up Python {% raw %}${{ matrix.python-version }}{% endraw %}
-        uses: actions/setup-python@v3
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
```

Comment:
For my information: why is this action needed? I believe `docker/build-push-action` already uses BuildKit, if that's the reason.